README.rst

```diff
@@ -17,39 +17,39 @@ future users of the library.
 
 For more details on new features please refer to the
 `release notes <https://github.com/sociomantic-tsunami/libdrizzle-redux/releases>`_
+Support guarantees
+------------------
+
+- Major branch development period: 6 months
+- Maintained minor versions: 2 most recent
+
+Maintained Major Branches
+-------------------------
+
+.. csv-table::
+   :header: "Major", "Initial release date", "Supported until"
+   :widths: 5, 15, 12
+
+   `v5.x.x`_, `v5.2.0`_: 2017-03-31, 2017-05-01
```

Comment:
I wonder about the supported until date here. I would keep it as TBA until we have a `v6.x.x` released.